Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -94,11 +94,11 @@ For more information about suites and how to declare and customize them, see
 
 ### Convert setup and teardown functions
 
-In XCTest, code can be scheduled to run before and after a test using the
+In XCTest, you can schedule code to run before a test using the
 [`setUp()`](https://developer.apple.com/documentation/xctest/xctest/3856481-setup)
-and [`tearDown()`](https://developer.apple.com/documentation/xctest/xctest/3856482-teardown)
-family of functions. When writing tests using the testing library, implement
-`init()` and/or `deinit` instead:
+family of functions. When writing tests using the testing library, you can use
+`init()` for setup. Your suite's initializer can be async, throwing, or
```

Comment:
Let's explicitly use the `async` and `throws` keywords like we do for test functions here: https://github.com/apple/swift-testing/blob/main/Sources/Testing/Testing.docc/DefiningTests.md